setup.py

```diff
@@ -1,68 +1,22 @@
 import os
 from IPython.html.nbextensions import install_nbextension
+from IPython.html.services.config import ConfigManager
 from IPython.utils.path import locate_profile
 
 
 livereveal_dir = os.path.join(os.path.dirname(__file__), 'livereveal')
 
 
-custom_js_entry = """
-
-// to prevent timeout
-requirejs.config({
-    waitSeconds: 60
-});
-
-$([IPython.events]).on('app_initialized.NotebookApp', function(){
-
-     require(['nbextensions/livereveal/main'],function(livereveal){
-       // livereveal.parameters('theme', 'transition', 'fontsize', static_prefix);
-       //   * theme can be: simple, sky, beige, serif, solarized
-       //   (you will need aditional css for default, night, moon themes).
-       //   * transition can be: linear, zoom, fade, none
-       livereveal.parameters('simple', 'zoom');
-       console.log('Live reveal extension loaded correctly');
-     });
-
-});
-"""
-
-
-def install(use_symlink=False, profile='default'):
+def install(use_symlink=False, profile='default', enable=True):
     # Install the livereveal code.
     install_nbextension(livereveal_dir, symlink=use_symlink,
-                        overwrite=use_symlink)
+                        overwrite=use_symlink, user=True)
```

Comment:
we should add the possibility of a system-wide installation too... but we can discuss that later...